core/src/consensus.rs

```diff
@@ -2797,6 +2805,28 @@ pub mod test {
         assert_eq!(tower.root(), Some(replayed_root_slot));
     }
 
+    #[test]
+    fn test_adjust_lockouts_after_replay_all_not_found_even_if_rooted() {
```

Comment:
this is new test for https://github.com/solana-labs/solana/pull/10718/commits/929ca6d859b697d54753e1e4b85c04c85d7822a0#diff-d848cba681535238849e7e33ed32702bR879-R885